draft-ietf-core-href.md

```diff
@@ -1002,13 +1002,14 @@ authority
   character that a dot is.)
 
   {:aside}
-  > Implementations with scheme-specific knowledge MAY convert
-    individual elements by using the ToASCII procedure ({{Section 4.1 of
+  > As an implementation note, an implementation with scheme-specific
```

Comment:
Is there anything scheme specific to this aspect?

IIRC, the "scheme specific" was from the "An IRI with a scheme that is known to" of 3987, but this note is now just about cases where the hostname part goes into DNS.

Reply:
... which is scheme-specific knowledge.

I'll merge this now, but maybe we can have another round at further improving the *-specific wording later.